pytorch_blade/src/ltc/disc_compiler/passes/cluster.cpp

```diff
@@ -120,6 +143,36 @@ void CastGraphInputsToTensor(
   }
 }
 
+bool IsDiscFusable(const torch::jit::Node* node) {
```

Comment:
Duplicated with`isDiscSupports`?

Reply:
Yes, and I will remove `cluster.cpp/h` in the future.

Reply:
done.